settings.gradle

```diff
@@ -55,6 +55,7 @@ include "solr:packaging"
 include "solr:distribution"
 include "solr:docker"
 include "solr:prometheus-exporter"
+include "solr:modules:aws-secret-provider"
```

Comment:
can this be up in alphabetical order like the other modules?

Reply:
Done :)